doc/elements.md

```diff
@@ -88,6 +88,10 @@ Attribute | Type | Default | Description
 `atol` | number | 1e-8 | Absolute tolerance for `comparison="relabs"`.
 `digits` | integer | 2 | number of digits that must be correct for `comparison="sigfig"` or `comparison="decdig"`.
 `allow-complex` | boolean | False | Whether or not to allow complex numbers as answers. If the correct answer `ans` is a complex object, you should use `import prairielearn as pl` and `data['correct_answer'][answers-name] = pl.to_json(ans)`.
+`hide-help-text` | boolean | False | Hide question mark at the end of the input displaying required input parameters.
+`placeholder` | boolean | True | Show the placeholder text that shows the default comparison.
```

Comment:
I think I'd prefer `show-placeholder` to go along with the `[verb]-[noun]` naming of other boolean attributes ("allow complex", "hide-help-text", etc.)

Reply:
if we change this to `show-placeholder`, should we also change the attribute in `pl-string-input`? I think we should keep the element attributes as consistent as possible

Reply:
The difference here is that `placeholder` for `pl-string-input` actually specifies placeholder text, whereas this one is a boolean on/off switch. I think it's actually more consistent to name them different things.

Reply:
oh yes.. you are correct! makes sense to change to `show-placeholder` :-)